KalturaWowzaServer/src/main/java/com/kaltura/media/server/wowza/CuePointsManager.java

```diff
@@ -324,7 +324,10 @@ public void sendSyncPoint(final IMediaStream stream, String id, double time, dou
 		data.put(OFFSET_KEY, offset);
 		data.put(TIMESTAMP_KEY, time);
 
-		stream.sendDirect(CuePointsManager.PUBLIC_METADATA, data);
+		//This condition is due to huge duration time (invalid) in the first chunk after stop-start on FMLE.
+		if (stream.getPlayPackets().size() > 0) {
```

Comment:
We can't tell, it's closed source, we can check it, but it won't promise anything for future use cases or future Wowza versions.